### PR TITLE
Added formula for cromshell2 beta

### PR DESCRIPTION
**Pull request by bshifaw**

### CODE REVIEW

File: Formula/cromshell@2.0.0.beta.1.rb [hunk 1]
```diff
@@ -0,0 +1,228 @@
+class CromshellAT200Beta1 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python tool to interact with a Cromwell server"
+  homepage "https://github.com/broadinstitute/cromshell"
+  url "https://github.com/broadinstitute/cromshell/archive/refs/tags/2.0.0.beta.1.tar.gz"
+  sha256 "85b3b3baa1c780b8da0d25bbbd6f89fcc5a345e803549118c350a4db6178155b"
+  license "BSD-3-Clause"
+  depends_on "python@3.9"
```
Comment (lbergelson) [inline Formula/cromshell@2.0.0.beta.1.rb:9]: I think I asked about this last time, but is there a way of saying python 3.9+?  I can't remember...

**Review by lbergelson — COMMENTED** [review]
@bshifaw This looks good to me.  One question about the python version dependency.  Think we should drop the "1" from the beta specifier in the formula so people get automatic updates if we release future betas?

File: Formula/cromshell@2.0.0.beta.1.rb (hunk 1 above)
Comment (bshifaw) [reply Formula/cromshell@2.0.0.beta.1.rb:9]: I couldn't find a way to do that in the Homebrews documentation. Also, couldn't find an example of having a minimum version through a github search, the majority of other formulas specified the version as "python@3.9"

Comment (bshifaw) [reply Formula/cromshell@2.0.0.beta.1.rb:9]: (Also had an extensive talk about it with Chatgpt, haha)

**Comment by bshifaw** [comment]
If the 1 was dropped wouldn't the URL point to the wrong path at line 6?

**Comment by bshifaw** [comment]
@lbergelson ^

**Pull request merged**